package.json

```diff
@@ -20,6 +20,7 @@
     "@emotion/react": "^11.1.5",
     "@emotion/styled": "^11.3.0",
     "@mdx-js/mdx": "^2.0.0",
+    "@next/bundle-analyzer": "^13.5.3",
```

Comment:
Could we move this into the `DevDependencies`? I think the workflow should still work if we move it there and that way this wouldn't be required for the "production" build